packages/calcite-design-tokens/support/types/tokenTypes/fontWeight.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint @cspell/spellchecker: 0 */
+export enum FontWeight {
+  black = 900,
+  bold = 700,
+  buch = 400,
+  demi = 600,
+  demibold = 600,
```

Comment:
We could include the [`fonts` dictionary](https://cspell.org/docs/dictionaries/#miscellaneous-dictionaries) to help with some of these spell checker warnings.

Reply:
these are all coming from token-studio so I want to leave this list as-is for now